### PR TITLE
Correction of name of function setUserId for the node docs

### PR DESCRIPTION
**Pull request by atrizdawiz**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* The function for setting userId is named **setUserId**, _not_ setUserID, i.e. with a large D.
* Try calling the function with a large D and see your application crash.
* I'm using version 11.2.1 of the new relic agent, together with @newrelic/next version 0.7.0.

### CODE REVIEW

**Comment by bradleycamacho** [comment]
hi @atrizdawiz, thanks for the edit! I confirmed with our product team that the docs are correct as is: https://github.com/newrelic/node-newrelic/blob/4c30d977ca74421cfa396c2f86383212e40475e1/api.js#L1787

Did you run into a specific issue where your version worked?

File: src/content/docs/apm/agents/nodejs-agent/api-guides/nodejs-agent-api.mdx [hunk 1]
```diff
@@ -859,16 +859,16 @@ New Relic's Node.js agent includes additional API calls.
   </Collapser>
 
   <Collapser
-    id="setUserID"
-    title={<InlineCode>newrelic.setUserID(string)</InlineCode>}
+    id="setUserId"
```
Comment (bizob2828) [inline src/content/docs/apm/agents/nodejs-agent/api-guides/nodejs-agent-api.mdx:862]: this isn't correct. The API docs are (setUserID)[https://newrelic.github.io/node-newrelic/API.html#setUserID]

Comment (atrizdawiz) [reply src/content/docs/apm/agents/nodejs-agent/api-guides/nodejs-agent-api.mdx:862]: This is what I see on the newrelic object in the browser after initializing the script:
image

I.e. with a small d. 
We are using  newrelic: "11.9.0" and @newrelic/next: "0.7.0".

Comment (atrizdawiz) [reply src/content/docs/apm/agents/nodejs-agent/api-guides/nodejs-agent-api.mdx:862]: So maybe the browser function is spelled differently? A little cofusing.

**Pull request closed without merging**